ark/phenotyping/pixel_consensus_cluster.R

```diff
@@ -77,3 +82,10 @@ for (i in 1:length(fovs)) {
         print(i)
     }
 }
+
+# save the mapping from cluster to hCluster_cap
+print("Writing SOM to meta cluster mapping table")
+hClustLabeled <- as.data.table(hClust)
+hClustLabeled$cluster <- as.integer(rownames(hClustLabeled))
+hClustLabeles <- setnames(hClustLabeled, "hClust", "hCluster_cap")
```

Comment:
Probably want the names to be "cluster" and "hCluster_cap" since you're mapping the cluster number to hierarchical cluster number.

Reply:
So this line only renames `hClust` to `hCluster_cap`: the columns `cluster` and `hCluster_cap` are the final columns. Nevertheless, I'll rename the variable `hClust` to `hCluster_cap` so that when `as.data.table(hCluster_cap)` is called, it will actually name the column `hCluster_cap`.